File: backend/src/main/java/hu/szakdoga/backend/timetable/data/dto/TimetableDTO.java
package hu.szakdoga.backend.timetable.data.dto;

public class TimetableDTO {
    public final Long id;
    public final String name;
    public final Long userId;

    public TimetableDTO(Long id, String name, Long userId) {
        this.id = id;
        this.name = name;
        this.userId = userId;
    }
}

File: frontend/src/app/shared/model/timetable/dto/timetable.dto.ts
export class TimetableDto {
    public readonly id: number | null;
    public readonly name: string;
    public readonly userId: number;

    constructor(name: string, userId: number, id: number | null = null) {
        this.id = id;
        this.name = name;
        this.userId = userId;
    }
}
